containers/netty-http/src/main/java/org/glassfish/jersey/netty/httpserver/NettyHttpContainerProvider.java

```diff
@@ -58,33 +58,67 @@
      * Create and start Netty server.
      *
      * @param baseUri       base uri.
-     * @param configuration Jersey configuration.
+     * @param container     Jersey container.
      * @param sslContext    Netty SSL context (can be null).
      * @param block         when {@code true}, this method will block until the server is stopped. When {@code false}, the
      *                      execution will
      *                      end immediately after the server is started.
      * @return Netty channel instance.
      * @throws ProcessingException when there is an issue with creating new container.
      */
-    public static Channel createServer(final URI baseUri, final ResourceConfig configuration, SslContext sslContext,
+    public static Channel createServer(final URI baseUri, final NettyHttpContainer container, SslContext sslContext,
```

Comment:
You are making backwards incompatible change. Why?

Reply:
Actually I didn't. I *added an additional method*, but the diff viewer displays this in a rather confusing way, so you got misleaded.

The original signature still exists see, line 156:
https://github.com/eclipse-ee4j/jersey/blob/fa9d0fe2d39be5e475035ac626a1bf19e0290753/containers/netty-http/src/main/java/org/glassfish/jersey/netty/httpserver/NettyHttpContainerProvider.java#L156